packages/front-end/components/Experiment/AnalysisSettingsBar.tsx

```diff
@@ -545,6 +545,14 @@ function isDifferentStringArray(
   if (val1.length !== val2.length) return true;
   return val1.some((v) => !val2.includes(v));
 }
+function isStringArrayMissingElements(
+  strings: string[] = [],
+  elements: string[] = []
+) {
+  if (!elements) return false;
```

Comment:
not needed as elements is always an array

Reply:
I disagree? if elements is an empty array we can return right away.